pkg/handler/processor/guesser/type_spdx.go

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package guesser
+
+import (
+	"bytes"
+
+	"github.com/guacsec/guac/pkg/handler/processor"
+	spdx_json "github.com/spdx/tools-golang/json"
+)
+
+type spdxTypeGuesser struct{}
```

Comment:
register guesser  in https://github.com/guacsec/guac/blob/main/pkg/handler/processor/guesser/type_guesser.go as well!